### ISSUE TITLE
Logs not found 

### ISSUE THREAD

**Issue by sukhmeet2390** [jorgebastida/awslogs#143]
`awslogs get /aws/lambda/xxxxxx-4c5e-9a0f-839d4466dc93 2017/07/18/[\$LATEST]6e171be7a8784a199630944201f5948b`
Returns nothing 
but 
`aws logs get-log-events --log-group-name /aws/lambda/xxxxx-4c5e-9a0f-839d4466dc93 --log-stream-name 2017/07/18/[\$LATEST]6e171be7a8784a199630944201f5948b`
returns the logs

**Comment by jorgebastida** [comment]
Have you tried ``awslogs get "/aws/lambda/xxxxxx-4c5e-9a0f-839d4466dc93 2017/07/18/[$LATEST]6e171be7a8784a199630944201f5948b"``

**Comment by Martijn02** [comment]
I'm seeing this issue as well. It has to do with #96, but the fix implemented for that ticket doesn't fix this issue.

The problem is caused by the `describe_log_streams `api returning stale timestamps for `lastEventTimestamp`. I have a couple of logs that produce a small amount of events every couple of minutes (a syslog). If I look in the AWS CloudWatch console, and open the streams for a group. I see a couple if streams that have a "Last Event Time" that is approximately 30 minutes behind. When I open that stream, I see that new logs have been added just a few minutes ago,

With the fix for #96, this behaviour was more or less fixed for the `awslogs streams [groupname]` functionality. But when you do `awslogs get [groupname] [streamname_regex]` the streams containing valid logs, are being filtered out by the `get_streams` function which uses the same value for self.start as for the `filter_log_events` api call. 

Result is that, when using the default 5m start time, the streams are never included in the `filter_log_events` api call, and logs in those streams do not appear in the results.

With longer time periods (1h) this behaviour doesn't show because the AWS `lastEventTimestamp` is never that far behind. 

I have fixed (very crudely) this by doing this:
```
@@ -221,7 +221,7 @@
     def get_streams(self, log_group_name=None):
         """Returns available CloudWatch logs streams in ``log_group_name``."""
         kwargs = {'logGroupName': log_group_name or self.log_group_name}
-        window_start = self.start - 3600000 or 0
+        window_start = self.start or 0
         window_end = self.end or sys.float_info.max

         paginator = self.client.get_paginator('describe_log_streams')
```

That makes the get_streams function return streams that have been updated an hour earlier than the specified start time. But this behaviour is not necessary for window_start's that are more than 1 hour in the past, and it doubles the effect of the solution for #96.

On the positive side, this hack does not show to many events, as the `filter_log_events` api is still being called with the correct start time. It just included a couple of extra stream names in the search for events.

**Comment by jorgebastida** [comment]
https://github.com/jorgebastida/awslogs/pull/199 should fix this issue. These changes will be released as part of ``0.11.0``.
